src/odin_gpib/keithley2410.py

```diff
@@ -27,11 +27,13 @@ def __init__(self, device, ident, lock):
             self.output_state = True
         if ("0" in output_init_state):
             self.output_state = False
-        
+
         print("OUTPUT: ",self.output_state)
 
-        self.device_control_enable = True 
-        self.ramping_flag = False 
+        self.device_control_enable = True
+        self.ramping_flag = False
+        self.write(':DISP:WINDOW1:TEXT:STAT 0')
```

Comment:
Can these calls to reset the display text on initialisation in both 2410 & 2510 classes be refactored into the base class, or call `self.set_identify(False)` ??

Reply:
addressed